hive3/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergInputFormat.java

```diff
@@ -63,19 +61,15 @@ public class HiveIcebergInputFormat extends MapredIcebergInputFormat<Record>
       HIVE_VECTORIZED_RECORDREADER_CTOR;
 
   static {
```

Comment:
I don't think the `static` block is necessary anymore, meaning you can inline this into L61